src/main/java/website/frontrow/artificial/intelligence/PathCalculation.java

```diff
@@ -0,0 +1,120 @@
+package website.frontrow.artificial.intelligence;
+
+import java.util.ArrayList;
+
+import website.frontrow.board.Mover;
+import website.frontrow.board.Enemy;
+import website.frontrow.level.Level;
+import website.frontrow.util.Point;
+
+/**
+ * This class can calculate the best path for an AI to take 
+ * given some info. //TODO: fix comment.
+ * 
+ * This method has no use for a constructor because it always returns a path.
+ */
+@SuppressWarnings("checkstyle:hideutilityclassconstructor")
+
+public class PathCalculation 
+{
+	private static ArrayList<Integer> goal;
+	private static Point location;
+	
+	/**
+	 * Method that returns a list of all relative positions of enemies and the player.
+	 * On the X axis
+	 * @param player the player unit.
+	 * @param enemies list of all enemy units.
+	 * @param level the current level.
+	 * @return a list of all relative positions of enemies and the player, on the X axis.
+	 */
+	public static ArrayList<Integer> calculateXPath(
+			Mover player, ArrayList<Enemy> enemies, Level level)
+	{
+		location = player.getLocation();
+		goal = new ArrayList<Integer>();
+		
+		for(int i = 0; i < enemies.size(); i++)
```

Comment:
You are also looping over the same array twice, you can merge these two loops together.